### PR TITLE
added projects in projects sec

### PR DESCRIPTION
**Pull request by Jaggu-123**
screenshot 142
screenshot 143
screenshot 144
screenshot 145
screenshot 146
screenshot 147

### CODE REVIEW

**Comment by ananya-agrawal** [comment]
@Jaggu-123 please provide link to gh-page

File: data/projects.json [hunk 1]
```diff
@@ -1,12 +1,172 @@
...
-			"mentors": "Team OpenCode",
-			"about": "A collaborative website created by particpants of opencode'18.",
-			"lang": "HTML, JavaScript, CSS",
-			"github": "https://github.com/opencodeiiita/Opencode-Collaborative-19"
-		}
-	]
+    "projects": [
+        {
+            "name": "OpenCode Collaborative",
+            "imageurl": "assets/img/participants/zerefwayne.jpg",
```
Comment (pradeepgangwar) [inline data/projects.json]: Add OpenCode logo here.

File: data/projects.json [hunk 2]
```diff
@@ -1,12 +1,172 @@
...
+            "imageurl": "assets/img/projects/tracking.png",
+            "mentors": "Team OpenCode",
+            "about": "An expense tracking app ade using React.",
+            "lang": "React Redux Bootstrap",
+            "github": "https://github.com/zerefwayne/expensify"
+        },
+        {
+            "name": "Selena-Gomez-Discography",
+            "imageurl": "assets/img/projects/music.jpg",
+            "mentors": "Team OpenCode",
```
Comment (pradeepgangwar) [inline data/projects.json]: Add mentor name here. This is not managed by team opencode.

File: data/projects.json [hunk 3]
```diff
@@ -1,12 +1,172 @@
...
+            "about": "One Pager containing all of the albums, songs, singles ",
+            "lang": "HTML CSS JS",
+            "github": "https://github.com/OrionStar25/Selena-Gomez-Discography"
+        },
+        {
+            "name": "File-Creator-CLI",
+            "imageurl": "assets/img/projects/filecreator.png",
+            "mentors": "Team OpenCode",
```
Comment (pradeepgangwar) [inline data/projects.json]: Same as above. Remove all such insatances.

**Pull request merged**